ServerCore/Pages/Shared/_EventNavigationPartial.cshtml

```diff
@@ -127,7 +127,7 @@
             @if (Event?.AllowBlazor == true && Event?.EphemeralHackKillNotifications != true)
             {
                 <a class="nav-link position-relative" asp-page="/Events/Notifications">
-                    <span class="fab text-white">✉️</span>
+                    <span class="fab text-white">✉🔔</span>
```

Comment:
Why do two of the lines in this PR have ✉🔔 rather than just 🔔?

Reply:
Argh, working too fast will fix

Reply:
Should be fixed with newly-deployed #1183.